dataflow/src/main/java/org/checkerframework/dataflow/reachingdefinitions/ReachingDefinitionsValue.java

```diff
@@ -0,0 +1,51 @@
+package org.checkerframework.dataflow.reachingdefinitions;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.cfg.node.AssignmentNode;
+import org.checkerframework.javacutil.BugInCF;
+
+/**
+ * A reaching definitions (which is represented by a node) wrapper turning node into abstract value.
+ */
+public class ReachingDefinitionsValue implements AbstractValue<ReachingDefinitionsValue> {
+
+    /**
+     * A reaching definition is represented by a node, which can be a {@link
+     * org.checkerframework.dataflow.cfg.node.AssignmentNode}.
+     */
+    protected final AssignmentNode def;
+
+    @Override
+    public ReachingDefinitionsValue leastUpperBound(ReachingDefinitionsValue other) {
+        throw new BugInCF("lub of reachDef get called!");
```

Comment:
See https://github.com/eisop/checker-framework/pull/177/files#r832725570 and
https://github.com/eisop/checker-framework/pull/177/files#r832726176

Reply:
Should we solve it in this pr?

Reply:
Solved, thanks!